### ISSUE TITLE
Debian lintian errors for libadldap

### ISSUE THREAD

**Issue by yurtpage** [altlinux/admc#434]
When building a debian package with `debuild` it runs linitan to check for errors and it shows a few warnings:

    lacks-ldconfig-trigger usr/lib/x86_64-linux-gnu/libadldap.so
    package-name-doesnt-match-sonames libadldap
    shared-library-lacks-version usr/lib/x86_64-linux-gnu/libadldap.so libadldap.so

As far I understood the libadldap.so is not intended to be used by other software but this is just something internal for the admc itself.
So this is not a "shared" library but rather a plugin. Still the linitan thinks that this may be a shared library that should have pkgconfig triggers etc.
Here what I found how to fix the first warning:
https://gitlab.kitware.com/cmake/cmake/-/issues/21834

Still not sure it this is needed. Can you please clarify this or close the ticket.

**Comment by august-alt** [comment]
Yes, `libadldap` is an internal `ADMC` plugin and is not intended to be used in any other application.
